test/CloudNative.CloudEvents.UnitTests/Avro/Helpers/FakeGenericRecordSerializer.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Avro;
+using Avro.Generic;
+using CloudNative.CloudEvents.Avro.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace CloudNative.CloudEvents.UnitTests.Avro.Helpers;
+
+internal class FakeGenericRecordSerializer : IGenericRecordSerializer
+{
+    public byte[]? SerializeResponse { get; private set; }
+    public GenericRecord DeserializeResponse { get; private set; }
+    public int DeserializeCalls { get; private set; } = 0;
+    public int SerializeCalls { get; private set; } = 0;
+
+    public FakeGenericRecordSerializer()
+    {
+        DeserializeResponse = new GenericRecord(ParseEmbeddedSchema());
```

Comment:
With the schema exposed via a property, we don't need the schema parsing code or the change to the project file.

Reply:
Suggestion applied, I've removed the unnecessary schema parsing